app/jobs/articles/score_calc_job.rb

```diff
@@ -9,6 +9,7 @@ def perform(article_id)
       article.update_columns(score: article.reactions.sum(:points),
                              hotness_score: BlackBox.article_hotness_score(article),
                              spaminess_rating: BlackBox.calculate_spaminess(article))
+      article&.user&.calculate_score
```

Comment:
We know the article is present bc of the above line so I dont think we need the first safety operator. What is the reason for the second one? When would an article not have a user?